src/app/shared/issue-pr-card/issue-pr-card-reviewers/issue-pr-card-reviewers.component.html

```diff
@@ -0,0 +1,19 @@
+<div
+  class="reviewers"
+  *ngIf="this.reviews"
+  (mouseenter)="isHovered = true"
+  (mouseleave)="isHovered = false"
+  [style.min-width.px]="this.containerWidth"
+>
+  <ng-container *ngFor="let review of this.reviews">
+    <img
+      src="{{ this.review.avatarUrl }}"
+      alt=""
+      width="{{ this.AVATAR_WIDTH }}"
+      class="reviewer-avatar"
+      [class.expanded]="isHovered === true"
+      [class.collapsed]="isHovered === false"
+      matTooltip="{{ review.toText() }}"
+    />
```

Comment:
```suggestion
    <img
      [src]="this.review.avatarUrl"
      alt=""
      [width]="this.AVATAR_WIDTH"
      class="reviewer-avatar"
      [class.expanded]="isHovered"
      [class.collapsed]="!isHovered"
      [matTooltip]="review.toText()"
    />
```

Let's reduce unnecessary string interpolation in attributes